lighthouse-core/test/lib/proto-preprocessor-test.js

```diff
@@ -27,6 +27,30 @@ describe('processing for proto', () => {
     expect(JSON.parse(output)).toMatchObject(expectation);
   });
 
+  it('cleans up default runtimeErrors', () => {
+    const input = {
+      'runtimeError': {
+        'code': 'NO_ERROR',
+      },
+    };
+
+    const output = processForProto(JSON.stringify(input));
+
+    expect({}).toMatchObject(JSON.parse(output));
```

Comment:
I think you meant for the output to be inside `expect()` :)

Reply:
Oh I see what it's doing. I think `toMatchObject` is just not right for the assertion we're trying to make here then.

It's basically [not.toHaveProperty('runtimeError')](https://jestjs.io/docs/en/expect#tohavepropertykeypath-value) we're checking for right?

Reply:
Good call!  That's what I was looking for!